internal/grpc/proto/service.pb.go

```diff
@@ -0,0 +1,217 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Auto-generated code should not be committed:
- le'ts add this to .gitignore
- let's update the Makefile to
  - install the tooling required to generate the code
  - generate the code before building.
@ricardo6142dh I can help you with that, let's pair.

Reply:
Removed and added to gitignore